04 - Guides, Workflows, & Courses/for Theme Designers.md

```diff
@@ -31,6 +31,7 @@ This note collects resources and guides for beginner and expert theme designers
 - [[Using Visual Studio Code for Theme Development]] by [[damiankorcz|Damian Korcz]]
 - [[Graph view customization|Styling the graph view]] 
 - [[Obsidian Design System Community File]] A community built, component and styles library in [[Figma]] for reference and use while designing user interfaces for Obsidian
+- [How To Work with Obsidian 1.0's new CSS Variables](https://www.youtube.com/watch?v=yl0pvIRTWWo)  by [[chrisgrieser|pseudometa]]
```

Comment:
It's OK as it is, but how would you feel about moving it up to the 'Create a Custom Theme in Obsidian' link above and giving it the same movie icon - for consistency and to make it clear it's a video.

Reply:
did so in the last commit. going through the list, I noticed that the whole page could really use some better organization, so I re-ordered the list in a more reasonable manner. (no new content added)

Reply:
OK I can't review that in the website. The diffs are huge now.
Will look at it later on.

Reply:
Also, the summary in the PR no longer describes the change.

Reply:
oh right. edited the PR title to reflect the changes